docs/user/01-01-keda-module.md

```diff
@@ -0,0 +1,34 @@
+# Keda Module
```

Comment:
This file looks like a main landing page for users?
Should it be renamed to`readme.md` so that it renders nicely in github when sb arrives via such link ?
`https://github.com/kyma-project/keda-manager/tree/main/docs/user`